packages/changelog-extract/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@brisk-docs/changelog-extract",
+  "version": "1.0.0",
+  "description": "This package is used for extracting changelogs from your projects",
+  "main": "index.js",
+  "repository": " git@bitbucket.org:atlassian/changelog-extract.git",
+  "author": "Aparna Raj",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
```

Comment:
Can we extract the jest config out into a `jest.config.js` file? I try to keep this kind of config out of the `package.json` where possible.